actions/publisher/action.yml

```diff
@@ -0,0 +1,111 @@
+name: "Build and publish the distribution to the PyPI server"
+description: "Build and publish PyPi wheel using tox for ML repos e.g. sparseml, compressed-tensors"
+
+inputs:
+  publish_pypi_internal:
+    description: "Publish the distribution to the internal PyPI server"
+    required: true
+
+  publish_pypi:
+    description: "Publish the distribution to the pypi.org"
+    required: true
+
+  build_number:
+    description: "Typically the PR number or the Run ID as a fallback solution"
+    required: false
+    default: ${{ github.event.pull_request.number || github.run_id }}
+
+  prefix:
```

Comment:
Since we're adding it after the name, should be suffix.

Reply:
oh.. true :)